controls/radwatermarktextbox/getting-started.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Getting Started
+page_title: Getting Started
+description: Getting Started
+slug: watermarktextbox-getting-started
+tags: getting,started
+published: True
+position: 1
+---
+
+# Getting Started
+
+This tutorial will walk you through the required steps for using __RadWatermarkTextBox__. 
+
+* [Assembly References](#assembly-references)
+
+## Assembly References
+
+In order to use __RadWatermarkTextBox__ in your application, you need to add references to the following assembly:
+
+* __Telerik.Windows.Controls__
+
+## Adding RadWatermarkTextBox to the Project 
+
+#### __[XAML] Example 1: Defining RadWatermarkTextBox__
+{{region radwatermarktextbox-gettingstarted_0}}
+	<telerik:RadWatermarkTextBox/>
+{{endregion}}
+
+#### __Figure 1: Defining RadWatermarkTextBox__
+![](images/RadWatermarkTextBox_GettingStarted_01.png)
+
+> __RadWatermarkTextBox__ supports setting its __WatermarkContent__ or __WatermarkTemplate__ for visualizing the needed information. These mechanisms are covered in details in the [Features]({%slug radwatermarktextbox-features%}) topic.

```

Comment:
You ca move this to a new section. For example "Setting the Watermark". The section could contain the information from the current sentence + a code snippet that shows how to set the content and the template + images show the end result.